src/components/CurrentUser/CurrentUser.jsx

```diff
@@ -1,12 +1,61 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 import './CurrentUser.scss';
+import { userFromServer } from '../../api/todos';
 
-export const CurrentUser = () => (
-  <div className="CurrentUser">
-    <h2 className="CurrentUser__title"><span>Selected user: 2</span></h2>
+export class CurrentUser extends React.Component {
+  state = {
+    selectedUser: {},
+  }
 
-    <h3 className="CurrentUser__name">Ervin Howell</h3>
-    <p className="CurrentUser__email">Shanna@melissa.tv</p>
-    <p className="CurrentUser__phone">010-692-6593 x09125</p>
-  </div>
-);
+  componentDidMount() {
+    const { userId } = this.props;
+
+    userFromServer(userId)
+      .then(result => this.setState({
+        selectedUser: result,
+      }));
+  }
+
+  componentDidUpdate(prevProps) {
+    const { userId } = this.props;
+
+    if (prevProps.userId === userId) {
+      return;
+    }
+
+    userFromServer(userId)
+      .then(result => this.setState({
+        selectedUser: result,
+      }));
+  }
+
+  render() {
+    const { userId, clear } = this.props;
+    const { selectedUser } = this.state;
+
+    return (
+      <div className="CurrentUser">
+        <h2 className="CurrentUser__title">
+          <span>{`Selected user: ${userId}`}</span>
+        </h2>
+
+        <h3 className="CurrentUser__name">{selectedUser.name}</h3>
+        <p className="CurrentUser__email">{selectedUser.email}</p>
+        <p className="CurrentUser__phone">{selectedUser.phone}</p>
+        <button
+          type="button"
+          onClick={() => clear()}
```

Comment:
```suggestion
          onClick={clear}
```
what does `clear` means? Name it `clearSelection` or smth like that